<?xml version="1.0" encoding="UTF-8"?>
<launch>
	 <arg name="aruco_marker_id" value="13" />
  <node pkg="tf" type="static_transform_publisher" name="aruco_link_broadcaster" args="0.003 -0.23 0 1.57 0 0 ar_marker_$(arg aruco_marker_id) ground_link 100" /> 

  <arg name="cam_img_raw" value="/camera/color/image_raw" />
  <arg name="cam_info" value="/camera/color/camera_info" />
 	<arg name="output_frame" value="camera_color_optical_frame" />
  <!-- ar track alvar -->
	<node pkg="ar_track_alvar" type="findMarkerBundlesNoKinect" name="ar_track_bundle" args="4.5 0.08 0.2 $(arg cam_img_raw) $(arg cam_info) $(arg output_frame) $(find icclab_grasping_niryo)/marker_bundles/bundle9to17.xml" respawn="true" />
<!--  <node pkg="ar_track_alvar" type="individualMarkersNoKinect" name="ar_tracker" args="4.5 0.01 0.2 $(arg cam_img_raw) $(arg cam_info) $(arg output_frame)" respawn="true" />-->
    
  <node name="camera_calibration" type="calibrate_camera.py" pkg="icclab_grasping_niryo" output="screen"/>
</launch>
